src/crimson/osd/osd_operations/snaptrim_event.cc

```diff
@@ -408,10 +408,13 @@ SnapTrimObjSubEvent::remove_or_update(
       }
       // Stats reporting - Set number of objects trimmed
       if (num_objects_before_trim > delta_stats.num_objects) {
-	//int64_t num_objects_trimmed =
-	//  num_objects_before_trim - delta_stats.num_objects;
-	//add_objects_trimmed_count(num_objects_trimmed);
+	      int64_t num_objects_trimmed =
+	        num_objects_before_trim - delta_stats.num_objects;
```

Comment:
`num_objects_before_trim` is not an entirely correct name since `delta_stats` is local to each `SnapTrimObjSubEvent` and is zeroed. I think that we can drop it and replace with:

```
if (delta_stats.num_objects < 0) {
  int64_t num_objects_trimmed = std::abs(delta_stats.num_objects);
}